static/app/components/searchSyntax/grammar.pegjs

```diff
@@ -197,12 +197,12 @@ text_filter
 
 // Filter keys
 key
-  = value:[a-zA-Z0-9_.-]+ {
+  = value:[a-zA-Z0-9_.@-]+ {
```

Comment:
🤔 What about `+` in emails. Actually there's a lot of stuff that could be in an email 😬

Reply:
I'll add `+` too, but there are a lot of characters here we gotta support to make this technically correct, just going for a quickfix here since count_if will increase visibility on what's lacking on this regex. Will definitely re-visit later 👍

Reply:
I feel like.. I would almost rather not add them, since then we'll have to come and clean this stuff up.

Or maybe we should be explicit that this is a temporary thing?

Reply:
Hmmm that's a good point, let me see if i can change our function_args to a wildcard instead easily, and if not i'll add some comments explaining that the `@` is temporary

Reply:
https://github.com/getsentry/sentry/blob/a7b82be057fefc54aa9f5b0caa525f4f38f8c418/static/app/components/searchSyntax/grammar.pegjs#L219-L223 I would make this use `search_value` instead of `key`

Reply:
added new terms in the search grammar to handle quoted values in parameters in d26583b
`quoted_aggregate_param` is based on `quoted_value` since a single regex would consume the trailing `"`, wasn't able to re-use search_value here cause we don't want to replace escaped quotes with plain quotes in a function